### DISCUSSION TITLE
How to install 2.188.0 version? [total gem newby]

### DISCUSSION THREAD

**Discussion by emilaos** [fastlane/fastlane#19135] [Q&A]
Hi, as I understand, version 2.188.0 was just released to patch an Apple Id issue. Now, as a total newby to gems, I see I can either run **bundle update fastlane** or somehow install the sourcecode. Running the bundle update or similar commands don’t bump my local version.

What should I do? Wait?

Thanks!

**Comment by crazymanish** [comment]
Hi @emilaos 

First, do the uninstall and clean-up
- `sudo gem uninstall fastlane`
- `sudo gem cleanup fastlane`

then try `bundle install` or `bundle update`

**Reply by emilaos** [reply]
Thanks @crazymanish! I will give it a try...
